.gitignore

```diff
@@ -32,6 +32,7 @@ credentials.json
 /sites/
 /attributes.json
 /venv/
+/venv
```

Comment:
how is this different than the line above?

Reply:
I am using venv from another location (symlink) to prevent installing python dependencies each time.